airbyte-webapp/src/pages/OnboardingPage/useStepsConfig.tsx

```diff
@@ -13,7 +14,14 @@ const useStepsConfig = (
   setCurrentStep: (step: StepType) => void;
   steps: Array<{ name: JSX.Element; id: StepType }>;
 } => {
+  // exp-speedy-connection
+  const location = useLocation() as unknown as ILocationState<{ step: StepType }>;
```

Comment:
Need to type location state, as typescript will complain otherwise.
https://github.com/remix-run/history/issues/930